src/stream/src/executor/aggregation/agg_group.rs

```diff
@@ -189,20 +185,20 @@ impl<S: StateStore, Strtg: Strategy> EstimateSize for AggGroup<S, Strtg> {
     }
 }
 
-impl<S: StateStore, Strtg: Strategy> AggGroup<S, Strtg> {
+impl AggGroup {
     /// Create [`AggGroup`] for the given [`AggCall`]s and `group_key`.
     /// For [`crate::executor::SimpleAggExecutor`], the `group_key` should be `None`.
     #[allow(clippy::too_many_arguments)]
     pub async fn create(
         group_key: Option<GroupKey>,
         agg_calls: &[AggCall],
-        storages: &[AggStateStorage<S>],
-        result_table: &StateTable<S>,
+        storages: &[AggStateStorage<impl StateStore>],
+        result_table: &StateTable<impl StateStore>,
```

Comment:
I think we'd better keep `S` here? to ensure they have the same StateStore type, and also can make code clean and consistent with other executors.